### ISSUE TITLE
Screen Reader Accessibility is Missing

### ISSUE THREAD

**Issue by StormProductionsMusic** [rcmaehl/WhyNotWin11#313]
**Describe the bug**
Not accessible with a screen reader such as NVDA.

**To Reproduce**
1. Go to 'nvaccess.org' and install 'NVDA'
2. Launch the WhyNotWin11 app and try to see the results using the screen reader only

**Expected behavior**
The screen reader will let you hear the actual results.

**Screenshots**
No screenshots as it's about visually impaired and/or blind people.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 2.3.0.3

**Additional context**
There are a lot of visually impaired and/or blind users that are using Windows, and that would also like to know whether their systems are compatible with Windows 11.

**Comment by micwoj92** [comment]
Please see #258 and comments

**Comment by StormProductionsMusic** [comment]
> Please see #258 and comments

Unfortunately, there are too many comments in there, and due to my visual impairment, my screen reader can't load all of them.
Could you please tell me which is the most important in there? The shortcuts mentioned don't seem to work..

**Comment by micwoj92** [comment]
This pull request was to add JAWS, NVDA andSapi5. It is not approved yet. NVDA compatibility was removed due to us being uncertain (it has different license than this software). It still needs to be reviewed and approved by Robert, but you can already download and test it.

Link below is to download zip, feel free to test and share feedback. It will be very important. Thanks!

https://github.com/rcmaehl/WhyNotWin11/suites/3127806850/artifacts/71722850

**Comment by StormProductionsMusic** [comment]
No, I have no other screen readers - I only have NVDA. 
Please let me know when, and if NVDA compatibility will be a thing.

Thanks.

**Comment by rmcpantoja** [comment]
Hi @StormProductionsMusic
The first. Did you activate the enanced accessibility? Because when you open WhyNotWin11.au3 with the accessibility script included, when opening the prigram, the first thing is to ask yourself if you want to activate accessibility through an MSGBox. I don't know if you haven't done it, but I just tried it and it works for me with sapi5. The thing is, as @micwoj92 explained, originally my accessibility swite supported JAWS, NVDA and sapi 5 but the support for NVDA has been removed due to licensing issues, but it works great with sapi 5 for now until there is any decision with NVDA.
Here is the list of keybindings: This, I reiterate, works if Enhanced Accessibility is turned on.
CTRL + SHIFT + 1: VIOS type information and if it is compatible.
CTRL + SHIFT + 2: CPU information.
CTRL + SHIFT + 3: GPT information.
CTRL + SHIFT + 4: Installed ram information.
CTRL + SHIFT + 5: Secure boot information.
CTRL + SHIFT + 6: Hard disk information.
CTRL + SHIFT + 7: TPM version information and if it is compatible.
CTRL + SHIFT + c: Copy the last resource of the spoken system to the clipboard where it can be pasted anywhere.

**Comment by rmcpantoja** [comment]
Hi, I have solved the problems with the code and fixed everything.
Especially in the accessibility script, when creating the "config" folder that stores the settings in the script folder. What happens is that in the main script of the program the working directory was changed to the system directory, so privileges are needed, but it is already corrected and perhaps that is the problem that prevented the script from working.

**Comment by rmcpantoja** [comment]
Hi,
I don't know what commit you downloaded. The first included NVDA but there were still things to improve, but in the latter everything is ready, only that for now NVDA is no longer taken into account. It's weird what happens to you, since the accessibility script works perfectly for me when I press one of the 8 key combinations and it reads me with the voice I have in sapi5. Anyway, I will update the main script when there are changes to it.

**Comment by StormProductionsMusic** [comment]
Can you please give me the link to the version that you have and for which the accessibility script works as expected, so I can test it? Thanks.

**Comment by micwoj92** [comment]
@StormProductionsMusic 
This should be the latest version from before the NVDA was removed:
https://github.com/rcmaehl/WhyNotWin11/suites/3125228223/artifacts/71643119

**Comment by rmcpantoja** [comment]
No, the version that really works is this, that is, the last one that was compiled, which fixes a major bug with the configuration folder. I say important because that version, the program had the script directory change, and the configuration folder is used to save the reader's configurations, that is, what screen reader is being used. This is the one that really works: 
        https://github.com/rcmaehl/WhyNotWin11/suites/3136081985/artifacts/71976646

**Comment by micwoj92** [comment]
Yes, but that version that you linked doesnt support NVDA, does it?

**Comment by rmcpantoja** [comment]
> Yes, but that version that you linked doesnt support NVDA, does it?

NO, it is not compatible with NVDA, that is, it is the latest version that was built a few hours ago and it works definitely without any problem.
